earth_enterprise/BUILD_RHEL_Centos.md

```diff
@@ -0,0 +1,121 @@
+# GEE Build Setup for CentOS, RHEL 6 and RHEL 7     
+
+
+## Install git  
+    
+Install the system default version:
+    
+```
+sudo yum install git
+```
+    
+Or install the latest version from the IUS repo (recommended) [ [More Info] ](https://ius.io/GettingStarted/):
+```
+sudo yum install -y wget
+cd /tmp
+wget https://rhel7.iuscommunity.org/ius-release.rpm
+sudo yum install ius-release.rpm
+sudo yum install git2u-all
+```
+        
+        
+## Install git lfs
+   
+Execute the following commands
+    
+```
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
+sudo yum install git-lfs
+```
+
+
+
+## Enable additional repos
+Different yum repositoies need to be enabled for different distributions to install several required development tools.  
+
+### CentOS 7 
+```
+sudo yum install epel-release
+```
+
+### RHEL 7  
+
+```
+sudo subscription-manager repos --enable=rhel-7-server-extras-rpms
```

Comment:
Hm. What is the `server-extras-rpms` used for? I've been building without adding it.

Reply:
My understanding is that is RH's "official" version of " EPEL - used by customers on closed networks as part of their RH Satellite subscription.

Reply:
Maybe we should note that it is not required to build Open GEE.  It could be suggested as providing some benefit on some systems.

Reply:
Do you know if you can replace installing `epel-release` with enabling `rhel-7-server-extras-rpms` to build Open GEE?